### DISCUSSION TITLE
Not getting response in bot

### DISCUSSION THREAD

**Discussion by Nik720** [Samagra-Development/SamagraX-community#41] [Troubleshooting and Clarifications]
Hi @aayushkedia95 @ChakshuGautam

I have created bots using a netcore adapter but I am not getting a response in the bot. I have restarted the ODK server but still not getting a response.
Here are the bot details
```

Bot ID: 06801678-5ce9-470b-b2f0-eee070813774
formID: "global_form"

Bot ID: 563b8fca-7462-4bf3-819c-0111d377681d
formID: "sunbird-karnataka-bot"

Bot ID: 6895bd67-3c3d-436f-84c3-31cc03a2b2c3
formID: "sunbird-general-bot"
```

<img width="300" alt="Screenshot 2021-07-13 at 1 47 33 PM" src="https://user-images.githubusercontent.com/33539271/125417015-24b1410c-ba43-46ee-ac67-387f78ec3e98.png">

cc - @vinukumar-vs

**Comment by Nik720** [comment]
@aayushkedia95 @ChakshuGautam It is working fine now. Issue was I tried with samagra sandbox number but I have to try with netcore number and have to set UCI-bot instance 

cc - @vinukumar-vs

**Reply by aayushkedia95** [reply]
Sounds good, @Nik720
